tests/test_sims.py

```diff
@@ -898,6 +898,41 @@ def setUpClass(cls):
         # define random number generator
         cls.rng = np.random.default_rng(cls.seed)
 
+    def test_ER_np(self):
+        rng = np.random.default_rng(self.seed)
+
+        p = [[1, 0.4],
+             [0.4, 0.7]]
+
+        alpha = [1000, 0.05]
+
+        n = 100 
+        #Produce 1000 graphs with all nodes assigned to community 1, thus approximate
+        #1000 ER graphs with probability almost equal to 1.
+        graphs = []
+
+        for i in range(1000):
```

Comment:
and see if you can get away with this only being like 200? if not that's okay, as long as graph can be smaller

Reply:
I think this can be done withouth issues. I could possibly also do only 100 graphs. Will promptly verify and perform the change.